keps/sig-apps/0003-kubernetes-application-api.md

```diff
@@ -0,0 +1,1120 @@
+---
+kep-number: draft-20180115
+title: Kubernetes Application API
+authors:
+  - "@kow3ns"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
```

Comment:
We probably don't have to duplicate the owning-sig into the participating-sigs metadata.